packages/material-ui/package.json

```
@@ -50,8 +50,8 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.4",
-    "@material-ui/styles": "^4.11.4",
-    "@material-ui/system": "^4.11.3",
+    "@material-ui/styles": "^4.12.0",
```

We only changed the tests

```suggestion
    "@material-ui/styles": "^4.11.4",
```